### DISCUSSION TITLE
Function hooks with more info

### DISCUSSION THREAD

**Discussion by alexverdaguer** [supabase/supabase#7220] [Questions] [answered]
Hi,

I have a many to many relationship table called "registered_event" with 2 rows, event_id and user_id, both are foreign keys for users table and events table with all the information.

I have a function hook that sends an email to the user once a record is inserted, but with req.body.record I just have access to the event_id and user_id. Is there a way somehow to access to the detail of each table ? Or do I have to have all these rows in the "registered_event" table in order to do that ?

Thank you.

**Comment by GaryAustin1** [comment, marked as answer]
Yes, add to your table, or make a DB call to get the info from your code sending the email.  Function hooks currently do nothing with foreign keys, just send the row in the table.  
I glanced at https://github.com/supabase/pg_net/issues and don't see any suggestions to implement it.

**Reply by alexverdaguer** [reply]
Hi @GaryAustin1, you are right, because I'm using a nextjs API route I can fetch all the data with these 2 fields ;)

Thank you !
